README.md

```diff
@@ -684,6 +684,208 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is with _Stripe_ using `ACH`.
+When capturing a bank account the flow can be divided in two steps:
+
+- Capturing
+- Verification
+
+Once the **Capture** has been made, Stripe will deposit two random amounts on the bank account. These amounts have to be
+verified against (or in other words, *sent to*) Stripe. You can do the verification process directly using _BillForward.js_.
+
+In more details what happens is:
+- Capture a bank account using _BillForward.js_
+- Stripe deposits on the bank account two random amounts
+- A BillForward PaymentMethod with `state = 'Pending'` is created
+- Verify the two amounts through _BillForward.js_
+- If the amounts are correct, BillForward switches the PaymentMethod `state` to `Active`
+- The bank account is now verified and ready to go
+
+
+#### Capturing with form
+
+This step creates an unverified bank account on both Stripe and BillForward backends. First of all a _HTML_ form is needed:
+
+```html
+<form id="payment-form">
+    <span class="payment-errors"></span>
+    <span class="payment-success"></span>
+
+    <div class="form-row">
+        <label>
+            <span>Holder Name</span>
+            <input type="text" size="20" bf-data="holderName" value="John Doe"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Bank Account Name</span>
+            <input type="text" size="20" bf-data="bankAccountName" value="John Doe's Bank"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Routing Number</span>
+            <input type="text" size="10" bf-data="routingNumber" value="110000000"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Account Number</span>
+            <input type="text" size="10" bf-data="accountNumber" value="000123456789"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Account Holder Type</span>
+            <select bf-data="accountHolderType">
+                <option>individual</option>
+                <option>company</option>
+            </select>
+        </label>
+    </div>
+
+    <input type="hidden" bf-data="accountID" value="SET AS ACCOUNT ID" />
+
+    <button type="submit">Submit Payment</button>
+</form>
+```
+
+The form is comprised of the following fields:
+
+| Field name          | Description |
```

Comment:
table is nice. :)

could you please add an "Example" column with sample inputs for each field?